<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KLFLibEntryEditor</class>
 <widget class="QWidget" name="KLFLibEntryEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KLFLibEntryEditor</string>
  </property>
  <layout class="QGridLayout" name="lyt_KLFLibViewEntryEditor">
   <property name="margin">
    <number>1</number>
   </property>
   <item row="0" column="0">
    <widget class="QSplitter" name="splitEntryEditor">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QFrame" name="frmPreviews">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QGridLayout" name="lyt_frmPreviews">
       <item row="0" column="0">
        <widget class="QLabel" name="lblPreview">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>280</width>
           <height>90</height>
          </size>
         </property>
         <property name="pixmap">
          <pixmap>:/pix/pics/nopreview.png</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTextEdit" name="txtPreviewLatex">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>80</height>
          </size>
         </property>
         <property name="tabChangesFocus">
          <bool>true</bool>
         </property>
         <property name="undoRedoEnabled">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QToolBox" name="tabsEntry">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabCategoryTags">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>297</width>
         <height>250</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <attribute name="label">
        <string>Category and Tags</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="lblCategory">
          <property name="text">
           <string>&amp;Category:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>cbxCategory</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbxCategory">
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="autoCompletion">
           <bool>true</bool>
          </property>
          <property name="duplicatesEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="_2">
          <item>
           <spacer name="spacer2_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnUpdateCategory">
            <property name="text">
             <string>Update Category</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="lblTags">
          <property name="text">
           <string>Name / &amp;Tags:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>cbxTags</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbxTags">
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="autoCompletion">
           <bool>true</bool>
          </property>
          <property name="duplicatesEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="_3">
          <item>
           <spacer name="spacer2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnUpdateTags">
            <property name="text">
             <string>Update Tags</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabStyle">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>149</width>
         <height>65</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <attribute name="label">
        <string>Style</string>
       </attribute>
       <layout class="QVBoxLayout" name="_4">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="margin">
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="lblStylePreview">
          <property name="text">
           <string>[Style]</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::TextBrowserInteraction</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnRestoreStyle">
          <property name="text">
           <string>Restore This Style</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
